src/astar/shibuya-price.js

```diff
@@ -0,0 +1,26 @@
+import Keyring from '@polkadot/keyring';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import { chains } from '@oak-network/config';
+import { schedulePriceTask } from './price-common';
+
+const createTaskPayload = (astarApi) => astarApi.tx.system.remarkWithEvent('Hello world!');
```

Comment:
Please change this to the ethereumChecked.transaction extrinsic even if the proxyExecuted will fail. We should stop using the `system.remarkWithEvent`, which is never the real use case. For now, we only need to make sure we see the xcmQueue event on the other parachain.

Reply:
Fixed